README.md

```diff
@@ -23,12 +23,6 @@ React with typescript example
 Telescope theme inspired by https://github.com/NvChad/NvChad (You have to enable it via config variable)
 ![telescope-theme](https://user-images.githubusercontent.com/31720261/151669762-1470aa12-b6ff-47c1-a4e9-ec9b37e0eabe.png)
 
-#### Data from images:
-
-Font: **Jetbrains Mono patched with nerd fonts**
-
-Terminal: https://github.com/wez/wezterm
-
```

Comment:
Let's keep this in here, since people people will ask themselves this questions right after seeing the images, if we put them in the bottom is not clear that this info is from this pictures.

Reply:
It's true that is not very important, but it is a nice-to-have, since almost all the time(at least myself) people are wondering what font/colors/terminals are been used on the code screenshots they see. I don't see any reason to remove them from here, they are there merely as nice-to-have info.